File: Java/Casing.java
public class Casing extends Komponen
{
    private String tipeCasing;
    private int jumlahFan;

    public Casing()
    {
        this.tipeCasing = "";
        this.jumlahFan = 0;
    }

    public Casing(String tipeCasing, int jumlahFan, String merk, String nama)
    {   
        super(merk, nama);
        this.tipeCasing = tipeCasing;
        this.jumlahFan = jumlahFan;
    }

    public void set_tipeCasing(String tipeCasing)
    {
        this.tipeCasing = tipeCasing;
    }

    public void set_jumlahFan(int jumlahFan)
    {
        this.jumlahFan = jumlahFan;
    }

    public String get_tipeCasing()
    {
        return this.tipeCasing;
    }

    public int get_jumlahFan()
    {
        return this.jumlahFan;
    }
}

File: Python/Casing.py
from Komponen import Komponen

class Casing(Komponen):
    __tipeCasing = ""
    __jumlahFan = 0

    def __init__(self, tipeCasing, jumlahFan, merk, nama):
        super().__init__(merk, nama)
        self.__tipeCasing = tipeCasing
        self.__jumlahFan = jumlahFan

    def get_tipeCasing(self):
        return self.__tipeCasing
    
    def get_jumlahFan(self):
        return self.__jumlahFan
    
    def set_tipeCasing(self, tipeCasing):
        self.__tipeCasing = tipeCasing

    def set_jumlahFan(self, jumlahFan):
        self.__jumlahFan = jumlahFan